### PR TITLE
minor typo fix

### PR DESCRIPTION
**Pull request by harunaadoga**
Fixed a minor typo in the README file.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,96 +1,96 @@
-
 # Starlight: Fast Container Provisioning
 
 [![Docker Image](https://img.shields.io/github/actions/workflow/status/mc256/starlight/docker-image.yml?branch=master&label=Proxy%20Docker%20Image&logo=docker&logoColor=white&style=flat-square)](https://github.com/mc256/starlight/actions/docker-image.yml)
 [![Helm Chart](https://img.shields.io/github/actions/workflow/status/mc256/starlight/helm-chart.yml?branch=master&label=Proxy%20Helm%20Chart&logo=helm&logoColor=white&style=flat-square)](https://github.com/mc256/starlight/actions/helm-chart.yml)
 [![Debian Package](https://img.shields.io/github/actions/workflow/status/mc256/starlight/debian-package.yml?branch=master&label=Snapshotter%20Debian%20Package&logo=debian&logoColor=white&style=flat-square)](https://github.com/mc256/starlight/actions/debian-package.yml)
 [![Release](https://img.shields.io/github/v/release/mc256/starlight?label=starlight&style=flat-square)](https://github.com/mc256/starlight/releases)
-<img align="right" src="docs/images/provisioning-time-wan.png">
+`<img align="right" src="docs/images/provisioning-time-wan.png">`
```
Comment (mc256) [inline README.md:7]: We need this image! Could you check how it looks?

File: README.md [hunk 2]
```diff
@@ -1,96 +1,96 @@
...
+   configuring containerd to use the plugin,
+   and starting the Starlight snapshotter daemon
+   (you also need to tell the snapshotter the address of the proxy server).
+   `<br>`[Find out how to install **containerd** &amp; **Starlight snapshotter plugin** ➡️](https://github.com/mc256/starlight/blob/master/docs/starlight-snapshotter.md)
 3) Convert the container image to the **Starlight format** container image.
-   More specifically, the storage format of the compressed layers needs to be converted to the Starlight format and then the layers stored in the registry. 
-   The Starlight format is **backwards compatible** and almost the same size, so there is no need to store compressed layers twice. In other words, non-Starlight workers will descrompress Starlight images with no chanages.
+   More specifically, the storage format of the compressed layers needs to be converted to the Starlight format and then the layers stored in the registry.
+   The Starlight format is **backwards compatible** and almost the same size, so there is no need to store compressed layers twice. In other words, non-Starlight workers will descrompress Starlight images with no changes.
```
Comment (mc256) [inline README.md:78]: This is the only meaningful change...

**Review by mc256 — CHANGES_REQUESTED** [review]

**Review by mc256 — CHANGES_REQUESTED** [review]
Hi @harunaadoga, thank you for your contributions. GitHub has done the job of keeping the history of a file. Could you please remove `.history/*`? With just two typos that you want to fix and nothing more? And please review your changes in Github so that the Markdown renders correctly.

**Comment by harunaadoga** [comment]
> 

Done, please feel free to ignore this since it's just two typos. Thanks!

**Comment by mc256** [comment]
Thanks, I will resolve these typos in another PR.

**Pull request closed without merging**